Sources/RInAppMessaging/Constants.swift

```diff
@@ -45,6 +46,12 @@ internal enum Constants {
 
     enum Versions {
         static let sdkVersion = "8.1.0-snapshot"
+        static var rmcSdkVersion: String? {
```

Comment:
@SoumenRautray This is not a constant. I think it should not be here. Consider adding it to `BundleInfo`?

Reply:
I agree with `BundleInfo`